Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/github"
+
+module Homebrew
+  module_function
+
+  def dispatch_build_bottle_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `dispatch-build-bottle` [<options>] <formula> [<formula> ...]
+
+        Build bottles for these formulae with GitHub Actions.
+      EOS
+      flag   "--tap=",
+             description: "Target tap repository (default: `homebrew/core`)."
+      flag   "--issue=",
+             description: "If specified, post a comment to this issue number if the job fails."
+      flag   "--macos=",
+             description: "Version of macOS the bottle should be built for."
+      flag   "--workflow=",
+             description: "Dispatch specified workflow (default: `dispatch-build-bottle.yml`)."
+      switch "--upload",
+             description: "Upload built bottles to Bintray."
+    end
+  end
+
+  def dispatch_build_bottle
+    args = dispatch_build_bottle_args.parse
+
+    raise FormulaUnspecifiedError if args.named.empty?
+
+    odie "Must specify --macos option" unless args.macos
```

Comment:
Doesn't need to be in this PR but it would be useful probably to have a way of triggering this for all macOS versions (if you want to build new bottles for all three versions without a `revision` PR job).

Reply:
I can try to do this in the future if there is a demand for that. It will require workflow changes, as we will have to construct a build matrix first, to build on multiple OSes and also create a second job which downloads bottles artifact and then does the upload to Bintray.

Now that I'm thinking about it, I might split the uploading/publishing steps into a second job now (soon), as it can become clearer for people what is going on in the workflow file and it will be easier to maintain.

Reply:
> I can try to do this in the future if there is a demand for that.

There is, saw @fxcoudert asking for it the other day 😂.

👍🏻 on everything else, nice work again 👏🏻